### PR TITLE
fix(network): display of IPv6 address

### PR DESCRIPTION
**Pull request by jomority**
This pull request fixes the display of the IPv6 address and its CIDR in the network module.

### CODE REVIEW

File: src/modules/network.cpp [hunk 1]
```diff
@@ -451,18 +450,34 @@ void waybar::modules::Network::getInterfaceAddress() {
...
+          cidrRaw >>= 1;
+        }
+      } else {
+        ipaddr_ = inet_ntop(AF_INET6,
+                            &reinterpret_cast<struct sockaddr_in6 *>(ifa->ifa_addr)->sin6_addr,
+                            ipaddr,
+                            INET6_ADDRSTRLEN);
+        auto net_addr = reinterpret_cast<struct sockaddr_in6 *>(ifa->ifa_netmask);
+        netmask_ = inet_ntop(AF_INET6, &net_addr->sin6_addr, netmask, INET6_ADDRSTRLEN);
+        for(size_t i = 0; i < sizeof(net_addr->sin6_addr.s6_addr); ++i) {
```
Comment (Alexays) [inline src/modules/network.cpp]: Missing space after `for`

File: src/modules/network.cpp [hunk 2]
```diff
@@ -451,18 +450,34 @@ void waybar::modules::Network::getInterfaceAddress() {
...
-      cidrRaw = net_addr->sin_addr.s_addr;
       unsigned int cidr = 0;
-      while (cidrRaw) {
-        cidr += cidrRaw & 1;
-        cidrRaw >>= 1;
+      if (family_ == AF_INET) {
+        ipaddr_ = inet_ntop(AF_INET,
+                            &reinterpret_cast<struct sockaddr_in *>(ifa->ifa_addr)->sin_addr,
+                            ipaddr,
+                            INET6_ADDRSTRLEN);
```
Comment (Alexays) [inline src/modules/network.cpp]: May be cleaner to use the inet addrstrlen instead but it doesn't matter much

**Comment by Alexays** [comment]
Just add the missing space, otherwise LGTM, thanks ❤️

**Comment by jomority** [comment]
I added the space, changed the size to INET_ADDRSTRLEN for IPv4 and rebased on the current master. Thanks for the comments :)

**Comment by Alexays** [comment]
Thanks ❤️

**Pull request merged**